packages/devextreme/js/__internal/ui/chat/messagegroup.ts

```diff
@@ -216,18 +218,44 @@ class MessageGroup extends Widget<Properties> {
 
   _updateMessageEditedText($message: dxElementWrapper, isEdited = false): void {
     const $firstMessage = this._$messageBubbleContainer.find(`.${CHAT_MESSAGEBUBBLE_CLASS}`).first();
+    const removeWithAnimation = ($editedElement: dxElementWrapper): void => {
+      $editedElement.get(0).addEventListener('animationend', () => {
+        $editedElement.remove();
+      }, { once: true });
+
+      $editedElement.addClass(CHAT_MESSAGE_EDITED_HIDING_CLASS);
+    };
 
     if ($message.is($firstMessage)) {
       const items = this.option('items');
-      this._renderMessageGroupInformation(items[0]);
+      const $information = this.$element().find(`.${CHAT_MESSAGEGROUP_INFORMATION_CLASS}`);
```

Comment:
If I add new message and new group is created then after edit of the message 'Edited' status is not added

Reply:
Because message doesn't store the updated isEdited value when you render

Reply:
added arg to make sure we set up correct edited state for information element
https://github.com/DevExpress/DevExtreme/pull/29765/commits/e5fc333773465733a5903baebba18c90386edb08